packages/mask/src/plugins/Tips/SNSAdaptor/TipsEntranceDialog.tsx

```diff
@@ -0,0 +1,363 @@
+import { DialogContent, Button } from '@mui/material'
+import { useI18N } from '../../../utils'
+import { makeStyles, useCustomSnackbar } from '@masknet/theme'
+import { VerifyAlertLine } from './components/VerifyAlertLine'
+import { useCallback, useEffect, useState } from 'react'
+import { enableRequirement } from '../constants'
+import { WalletsByNetwork } from './components/WalletsByNetwork'
+import ActionButton from '../../../extension/options-page/DashboardComponents/ActionButton'
+import SettingView from './bodyViews/Setting'
+import WalletsView from './bodyViews/Wallets'
+import AddWalletView from './bodyViews/AddWallet'
+import { useProvedWallets } from '../hooks/useProvedWallets'
+import {
+    BindingProof,
+    ECKeyIdentifier,
+    NextIDAction,
+    NextIDPersonaBindings,
+    NextIDPlatform,
+    NextIdStorageInfo,
+    NextIDStoragePayload,
+} from '@masknet/shared-base'
+import Empty from './components/empty'
+import { getKvPayload, kvSet, useKvGet } from '../hooks/useKv'
+import { InjectedDialog, LoadingAnimation } from '@masknet/shared'
+import { useAsyncFn, useAsyncRetry } from 'react-use'
+import Services from '../../../extension/service'
+import { PluginId } from '@masknet/plugin-infra'
+import { NextIDProof } from '@masknet/web3-providers'
+import { isSameAddress, useAccount } from '@masknet/web3-shared-evm'
+import formatDateTime from 'date-fns/format'
+import { LoadingButton } from '@mui/lab'
+import { useRemoteControlledDialog } from '@masknet/shared-base-ui'
+import { WalletMessages } from '@masknet/plugin-wallet'
+import { cloneDeep } from 'lodash-unified'
+export interface TipsEntranceDialogProps {
+    open: boolean
+    onClose: () => void
+}
+const useStyles = makeStyles()((theme) => ({
+    walletBtn: {
+        fontSize: '14px',
+    },
+    alertBox: {
+        marginBottom: '20px',
+    },
+    actions: {
+        position: 'absolute',
+        bottom: 16,
+        width: 'calc( 100% - 32px)',
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        gap: theme.spacing(1.5),
+    },
+    dContent: {
+        height: 600,
+        position: 'relative',
+        boxSizing: 'border-box',
+    },
+    btnContainer: {
+        width: '100%',
+        display: 'flex',
+        flexDirection: 'row-reverse',
+    },
+    loading: {
+        position: 'absolute',
+        top: '50%',
+        left: '50%',
+        transform: 'translate(-50%,-50%)',
+    },
+}))
+
+enum BodyViewSteps {
+    main = 'Tips',
+    setting = 'Settings',
+    wallets = 'Wallets',
+    addWallet = 'Add wallet',
```

Comment:
Any better name for these steps?

```suggestion
enum BodyViewStep {
    Main = 'Tips',
    Settings = 'Settings',
    Wallets = 'Wallets',
    AddWallet = 'AddWallet',
```